web/massastation/src/pages/Index/Index.tsx

```diff
@@ -75,60 +75,56 @@ export function Index() {
     <>
       <div className="bg-primary text-f-primary pt-24">
         <h1 className="mas-banner mb-10"> {Intl.t('index.title-banner')}</h1>
-        <div className="overflow-auto h-[65vh]">
-          <div className="w-[70vw]">
```

Comment:
Be careful here... we must be aligned with the design on the other pages (store, search) in terms of layout, title and etc. Check if they are aligned...

Reply:
I added : ```<div className="overflow-auto h-[65vh]">```

the other line is not necessary, as the component must take up the entire width of the screen, like in Figma